Kernel/VirtIO/VirtIOConsole.cpp

```diff
@@ -85,23 +109,39 @@ KResultOr<size_t> VirtIOConsole::read(FileDescription&, u64, [[maybe_unused]] Us
     if (!size)
         return 0;
 
-    return 1;
+    return ErrnoCode::EPERM;
```

Comment:
You dont have to explicitly qualify with `ErrnoCode::`, but wouldnt ENOTSUP be more appropriate here?

Reply:
Good idea, I'll use that instead. :)